src/pages/iou/steps/MoneyRequstParticipantsPage/MoneyRequestParticipantsSelector.js

```diff
@@ -34,8 +34,8 @@ const propTypes = {
     /** The type of IOU report, i.e. bill, request, send */
     iouType: PropTypes.string.isRequired,
 
-    /** The current tab we have navigated to in the request modal. String that corresponds to the request type. */
-    selectedTab: PropTypes.oneOf([CONST.TAB.DISTANCE, CONST.TAB.MANUAL, CONST.TAB.SCAN]).isRequired,
+    /** Whether the money request is a distance request or not */
+    isDistanceRequest: PropTypes.bool,
```

Comment:
I moved that up to `MoneyRequestParticipantsPage` which is the parent component.

There, I'm checking if the money request is a distance request or not using the `selectedTab`. For this component, `isDisntaceRequest` is passed from `MoneyRequestParticipantsPage`

This is because `MoneyRequestParticipantsPage` also needs to know if the request is a distance request or not